### PR TITLE
PLAT-126332: Add UI tests for agate/SwitchItem

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
 - Added ui tests for SwitchItem

### Links
[//]: # (Related issues, references)
PLAT-126332

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]

**Review by jeonghee27 — APPROVED** [review]
LGTM

**Comment by adrian-cocoara-lgp** [comment]
Added unit tests for `SwitchItem`. More can be added when SwitchItem gets refactored

cc: @jeonghee27, @daniel-stoian-lgp, @stanca-pop-lgp

File: tests/ui/specs/SwitchItem/SwitchItem-specs.js [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+		describe('5-way', function () {
+			it('should be able to focus the item', function () {
+				prevSwitchItem.focus();
+				Page.spotlightDown();
+				expect(switchItem.self.isFocused()).to.be.true();
+			});
+		});
+
+		describe('pointer', function () {
+			it('should not select the item when clicked', function () {
```
Comment (MikyungKim) [inline tests/ui/specs/SwitchItem/SwitchItem-specs.js:169]: ?
```suggestion
			it('should not unselect the item when clicked', function () {
```

Comment (adrian-cocoara-lgp) [reply tests/ui/specs/SwitchItem/SwitchItem-specs.js:169]: Ups. Thanks for noticing! 👍

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**